src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -449,11 +449,38 @@ public GHProject createProject(String name, String body) throws IOException {
 
     /**
      * The enum Permission.
+     *
+     * @deprecated Use <code>RolePermission</code> instead, which allows for custom permissions/roles created in github
+     *             to be set.
      */
+    @Deprecated
     public enum Permission {
         ADMIN, MAINTAIN, PUSH, TRIAGE, PULL
     }
 
+    /**
+     * Permissions (roles) for teams and collaborators.
+     */
+    public static class RolePermission {
```

Comment:
I think maybe this class should be `GHTeam.RepositoryPermissions` from what I'm reading in the docs.